hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/tasks/NSSummaryTaskWithLegacy.java

```diff
@@ -245,9 +249,10 @@ public boolean reprocessWithLegacy(OMMetadataManager omMetadataManager) {
                     .setObjectID(keyInfo.getObjectID())
                     .setParentObjectID(keyInfo.getParentObjectID())
                     .build();
-            handlePutDirEvent(directoryInfo, nsSummaryMap);
+            handlePutDirEvent(directoryInfo, nsSummaryMap, null, 0L);
```

Comment:
for status with 1L or 0L, we can use constant / enum to have meaning

Reply:
Ok.